src/v0/destinations/marketo/config.js

```diff
@@ -27,9 +27,9 @@ const formatConfig = (destination) => ({
     false,
   ),
   customActivityPrimaryKeyMap: getHashFromArray(
```

Comment:
can we make it backward compatible ?
- if `rudderEventsMapping` is empty, continue inferring from `customActivityPrimaryKeyMap`

Reply:
@anantjain45823 can you check this

Reply:
Already raised a PR [here](https://github.com/rudderlabs/rudder-transformer/pull/2866)